api/descriptions/collections/post.md

```diff
@@ -1,4 +1,4 @@
-Creates a new collection.
+Creates a new collection. You need to either specify a value or initialize the resouces in the endpoint with a "*" wildcard to be able to use the collection.
```

Comment:
@ssugandh Couple of things:

1. Please use semantic lines breaks (i.e. one sentence per line). See the guidance here: https://sembr.org/

2. Please update this sentence to: 

   ```
   You must specify a value for all resources in the collection, even if it's just a wildcard `*`.
   ```

3. Add a new paragraph that describes why it's important to specify a value for all resources, and what happens if you don't do that.

   ```
   If you create a collection and specify values for only some resources, then ... <WHAT'S THE OUTCOME, WHY THIS IS IMPORTANT, HOW YOU CAN TELL YOU DID SOMETHING WRONG, HOW TO VALIDATE, ETC>
   ```